tests/test_api.py

```
@@ -1237,6 +1239,94 @@ async def test_get_event_smart_detect_track(protect_client: ProtectApiClient):
     )
 
 
+@pytest.mark.asyncio()
+async def test_smart_detect_track_license_plate_none(protect_client: ProtectApiClient):
```

Ruff (0.14.5)

Unused function argument: `protect_client`

(ARG001)